next-version/next-version.sh

```diff
@@ -57,6 +57,13 @@ fi
 major_notes=$(generate_release_notes "$MAJOR")
 minor_notes=$(generate_release_notes "$MINOR")
 
+if [ ! -d "$MAJOR" ]; then
```

Comment:
This should be a function (it can take two arguments, a possible path and a category name). And it should only complain if the path argument isn't empty (`-z`)